uint stops docker build

Possibly introduced by #85 

Output:

```make
1.172 In file included from /app/example/main.cpp:3:
1.172 /app/include/expresso/middleware/cacher.h:12:3: error: 'uint' does not name a type; did you mean 'int'?
1.172    12 |   uint maxAge;
1.172       |   ^~~~
1.172       |   int
1.172 /app/include/expresso/middleware/cacher.h:17:14: error: expected ')' before 'maxAge'
1.172    17 |   Cacher(uint maxAge = 3600, bool isPrivate = false);
1.172       |         ~    ^~~~~~~
1.172       |              )
1.229 In file included from /usr/include/c++/13.2.1/bits/shared_ptr_base.h:59,
1.229                  from /usr/include/c++/13.2.1/bits/shared_ptr.h:53,
1.229                  from /usr/include/c++/13.2.1/condition_variable:45,
1.229                  from /app/lib/nexus/include/nexus/pool.h:3,
1.229                  from /app/include/expresso/core/server.h:7,
1.229                  from /app/example/main.cpp:2:
1.229 /usr/include/c++/13.2.1/bits/unique_ptr.h: In instantiation of 'std::__detail::__unique_ptr_t<_Tp> std::make_unique(_Args&& ...) [with _Tp = expresso::middleware::Cacher; _Args = {int, bool}; __detail::__unique_ptr_t<_Tp> = __detail::__unique_ptr_t<expresso::middleware::Cacher>]':
1.229 /app/example/main.cpp:82:31:   required from here
1.229 /usr/include/c++/13.2.1/bits/unique_ptr.h:1070:30: error: no matching function for call to 'expresso::middleware::Cacher::Cacher(int, bool)'
1.229  1070 |     { return unique_ptr<_Tp>(new _Tp(std::forward<_Args>(__args)...)); }
1.229       |                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1.229 /app/include/expresso/middleware/cacher.h:9:7: note: candidate: 'expresso::middleware::Cacher::Cacher()'
1.229     9 | class Cacher : public Middleware {
1.229       |       ^~~~~~
1.229 /app/include/expresso/middleware/cacher.h:9:7: note:   candidate expects 0 arguments, 2 provided
1.229 /app/include/expresso/middleware/cacher.h:9:7: note: candidate: 'expresso::middleware::Cacher::Cacher(const expresso::middleware::Cacher&)'
1.229 /app/include/expresso/middleware/cacher.h:9:7: note:   candidate expects 1 argument, 2 provided
1.329 make[2]: *** [CMakeFiles/server.dir/build.make:76: CMakeFiles/server.dir/example/main.cpp.o] Error 1
1.329 make[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/server.dir/all] Error 2
1.330 make: *** [Makefile:91: all] Error 2
------
build.dockerfile:13
--------------------
  11 |     COPY CMakeLists.txt /app/CMakeLists.txt
  12 |     
  13 | >>> RUN mkdir build && cd build && cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_EXE_LINKER_FLAGS="-static" .. && make
  14 |     
  15 |     RUN chmod +x /app/build/server
--------------------
ERROR: failed to solve: process "/bin/sh -c mkdir build && cd build && cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_EXE_LINKER_FLAGS=\"-static\" .. && make" did not complete successfully: exit code: 2
```

--- a/include/expresso/middleware/cacher.h
+++ b/include/expresso/middleware/cacher.h
@@ -9,12 +9,12 @@ namespace middleware {
 class Cacher : public Middleware {
 private:
   bool isPrivate;
-  uint maxAge;
+  uint32_t maxAge;
 
   std::string serialized;
 
 public:
-  Cacher(uint maxAge = 3600, bool isPrivate = false);
+  Cacher(uint32_t maxAge = 3600, bool isPrivate = false);
   ~Cacher();
 
   bool use(expresso::core::Request &req,

--- a/src/middleware/cacher.cpp
+++ b/src/middleware/cacher.cpp
@@ -1,6 +1,6 @@
 #include <expresso/middleware/cacher.h>
 
-expresso::middleware::Cacher::Cacher(uint maxAge, bool isPrivate)
+expresso::middleware::Cacher::Cacher(uint32_t maxAge, bool isPrivate)
     : isPrivate(isPrivate), maxAge(maxAge) {
   this->serialized = "max-age=" + std::to_string(maxAge);
   if (isPrivate) {

